XpringKit/JavaScript/JavaScriptUtils.swift

```diff
@@ -121,4 +121,11 @@ internal class JavaScriptUtils {
     }
     return result.toString()
   }
+
+  /// Convert the given byte array to a hexadecimal string.
+  /// - Parameter bytes: An array of bytes
+  /// - Returns: An encoded hexadecimal string.
+  public func toHex(bytes: Data) -> String {
+    return "STUB toHex"
```

Comment:
Is this to be implemented in the future? Or is it just for testing purposes?

Reply:
This is to be implemented in the future.  You'll see an explicit note to @keefertaylor in this PR because I need help wiring up the `xpring-common-js` module (in which `toHex` already resides) to be exposed here in the Swift wrapper class to that module.

Reply:
There's actually already functionality to do this in xpringkit:
https://github.com/xpring-eng/XpringKit/blob/master/XpringKit/XRP/ByteArray%2BHex.swift#L3

This is an extension on an array where the type of the array is a `uint8`. In english, anytime you have an array of bytes a special `toHex()` method gets added. 

`Data` is just a wrapper for a `[Uint8]` type in swift. So you can convert between the two fairly easily:
```
let data = Data()
let bytes = [UInt8](data)
let hex = byteArray.toHex()
```

Aside, I usually call `Data` types `data` and `[Uint8]` types bytes, though this is ultimately just a personal preference.

Reply:
Awesome, also easy! This should be fixed and ready for review now.